torch/_dynamo/mutation_guard.py

```diff
@@ -117,15 +117,7 @@ def is_dynamic_nn_module(obj: Any, is_export: bool) -> bool:
         return True
     if hasattr(obj, "torchdynamo_force_dynamic"):
         return obj.torchdynamo_force_dynamic
-    # For export, we will have to fix
-    # 1) Input signature problem because params are lifted as inputs
-    # 2) nn module stack info changes
-    # 3) adjust failing tests
-    if (
-        isinstance(obj, torch.nn.Module)
-        and config.inline_inbuilt_nn_modules
-        and not is_export
-    ):
+    if isinstance(obj, torch.nn.Module) and config.inline_inbuilt_nn_modules:
```

Comment:
Use the flag `config_install_params_as_graph_attr` such that setting it to False gives us old behavior (allows us to not revert the PR, just set the flag to False).